eoscostarica/proton-affiliate
# feat(webapp): use rewards amount and expirations hours from smart contract table

### What does this PR do?

- Resolve #143

### Steps to test

1. make run
2. navigate to the home
3. check info page information for rewards and expiration time

#### CheckList
